zconf-ng.h.in

```
@@ -6,13 +6,8 @@
 #ifndef ZCONFNG_H
 #define ZCONFNG_H
 
-#if defined(_WINDOWS) && !defined(WINDOWS)
-#  define WINDOWS
-#endif
-#if defined(_WIN32) || defined(__WIN32__)
-#  ifndef WIN32
-#    define WIN32
-#  endif
+#if !defined(_WIN32) && defined(__WIN32__)
```

Borland C++ again

---

That's the original code. Just check and you'll understand the logic

---

The logic is to catch Borland C and MSYS, but the define is wrong as we should fail hard with Borland C...